public/app/features/templating/templateSrv.js

```diff
@@ -107,10 +111,15 @@ function (angular, _) {
       });
     };
 
-    this.getAllValue = function(variable) {
+    this.getAllValue = function(variable, format) {
       if (variable.allValue) {
         return variable.allValue;
       }
+
```

Comment:
Also when I choose a couple of values in variable, but then click on "Clear selection", I get batched values instead of "All" value.